Secrets/KeysExample.swift

```diff
@@ -4,3 +4,12 @@ struct Keys {
         static let clientKey = ""
     }
 }
+
+func swiftLintTest() {
+    let someForceCast = NSNumber() as! Int
+    let colonOnWrongSide :Int = 0 
+}
+
+let x = 1
+
+let x = 1
```

Comment:
Name Format Violation (Medium Severity): Variable name should be between 3 and 40 characters in length: 'x'